src/main/webapp/i18n/en/tutorExerciseDashboard.json

```diff
@@ -14,6 +14,8 @@
             "readGradingInstructions": "Read grading instructions",
             "startYourParticipation": "Start your participation to the exercise",
             "gradingInstructions": "Grading instructions",
+            "problemStatement": "Problem statement",
+            "sampleSolution": "Sample solution",
```

Comment:
```suggestion
            "sampleSolution": "Example solution",
```

Reply:
Please use consistently the term **Example solution** in English, because there are typically multiple correct solutions and unfortunately, students might think that the sample solution would be the only correct solution, which is not correct